isovar/translation.py

```diff
@@ -324,13 +324,22 @@ def find_mutant_amino_acid_interval(
     # rounding down since a change in the middle of a codon should count
     # toward the variant codons
     n_coding_nucleotides_before_variant = len(cdna_coding_prefix)
-    n_complete_prefix_codons = cdna_variant_start_offset // 3
+
+    # subtract off the number of nucleotides we dropped to shift the sequence
```

Comment:
I'm confused by the subtracting comment here; there is only integer division here.